projects/ruby-lang.org/package.yml

```diff
@@ -79,10 +79,21 @@ build:
     # weirdly files get put here and we can't figure out how to stop it
     - run: |
         if test -d pkgconfig; then rm -rf pkgconfig; fi
-        mv *-{{hw.platform}}*/* .
-        rmdir *-{{hw.platform}}*
+        if test -d *-{{hw.platform}}* ; then
+          mv *-{{hw.platform}}*/* .
+          rmdir *-{{hw.platform}}*
+        fi
       working-directory: ${{prefix}}/lib
       if: '>=2.6'
+    - run: |
+        if test *-{{hw.platform}}*/bin/ruby ; then
```

Comment:
this shouldn't ever run, because of line 83, i think? should line 87 be changed to `'>=2.6<3.4'`?

Reply:
they're actually a different `working-directory` in each clause. one is moving stuff in `lib/` and one is moving stuff at the prefix root

Reply:
the main reason i added the `test` clauses was because the bug here seems like ruby might have some mishandling of the `--disable-multiarch` option in configure.ac, which necessitates the need for those workarounds. if it got fixed upstream it would break the formula without those new conditionals.

i don't totally have the history here why that option is being used (maybe there's a good reason), but if it was possible to just not use those options it might make the formula a little less brittle. something for another PR, maybe.

Reply:
Ah, good catch. Yeah, some of these formulas get complicated.